From weiyi.huang@gmail.com  Sun Jan 25 14:43:14 2009
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Sun, 11 Jan 2009 17:25:13 +0800
Subject: USB: gadget: remove duplicated #include
To: gregkh@suse.de, dbrownell@users.sourceforge.net
Cc: linux-usb@vger.kernel.org
Message-ID: <20090111172243.0AB7.WEIYI.HUANG@gmail.com>


Removed duplicated #include in
  drivers/usb/gadget/ci13xxx_udc.c

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/ci13xxx_udc.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/gadget/ci13xxx_udc.c
+++ b/drivers/usb/gadget/ci13xxx_udc.c
@@ -56,7 +56,6 @@
 #include <linux/dma-mapping.h>
 #include <linux/init.h>
 #include <linux/interrupt.h>
-#include <linux/interrupt.h>
 #include <linux/io.h>
 #include <linux/irq.h>
 #include <linux/kernel.h>
